cmd/wasirun/main.go

```diff
@@ -55,6 +56,9 @@ OPTIONS:
    --non-blocking-stdio
       Enable non-blocking stdio
 
+   --wasi-http
+      Enable wasi-http client support
+
```

Comment:
I am tempted to suggest we enable by default, what do you think @chriso ?

Reply:
One of the nice things about the sockets extensions is that wasi-go can autodetect which one you're using in https://github.com/stealthrocket/wasi-go/blob/main/imports/extensions.go. It'd be nice to do the same thing with this host module. If the module requires the HTTP host imports, we'd automatically set up the right host module. If new versions of `wasi-http` come out in future then ideally we'd be able to detect which one is in use and set it up automatically.

How about something like `-http <auto|none|v1(|v2...)>` which defaults to `auto`?

Reply:
Done.